.github/workflows/deploy_tre.yml

```diff
@@ -51,6 +52,12 @@ jobs:
           echo "using id of: ${REFID} for GitHub Ref: ${GITHUB_REF}"
           echo "::set-output name=refid::${REFID}"
 
+          if [ ${GITHUB_REF} == 'refs/heads/main' ]; then
+            echo "::set-output name=dev_container_label::latest"
+          else
+            echo "::set-output name=dev_container_label::${REFID}"
+          fi
```

Comment:
If this in the script because you wanted to be more explicit compared with the conditional expressions I used in the other outputs of this job?

Reply:
I wanted it to be readable, but will put it inline if you prefer